config/initializers/assets.rb

```diff
@@ -11,4 +11,4 @@
 # application.js, application.scss, and all non-JS/CSS in app/assets folder
 # are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
-Rails.application.config.assets.precompile += %w( geocoding.js )
+Rails.application.config.assets.precompile += %w( geocoding.js comments.js )
```

Comment:
you can remove this comments.js form here. because you will include it in application.js